product_form_sale_link/models/product_product.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    sale_lines_count = fields.Integer(
+        compute="_compute_sale_lines_count", string="Sold"
+    )
+
```

Comment:
Can we change the string to "Sales" as referenced in the xml or "Sale Lines", because it is in conflict with another field. 

```
2024-04-02 09:00:13,970 249 WARNING odoo odoo.addons.base.models.ir_model: Two fields (sale_lines_count, sales_count) of product.template() have the same label: Sold. [Modules: product_form_sale_link and sale] 
2024-04-02 09:00:13,970 249 WARNING odoo odoo.addons.base.models.ir_model: Two fields (sale_lines_count, sales_count) of product.product() have the same label: Sold. [Modules: product_form_sale_link and sale] 
```